.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,86 @@
+<!--
 Thanks for your hard work, please ensure all items are complete before opening.
+-->
+## Checklist
 
-- [ ] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://github.com/cloudant/sync-android/blob/master/DCO1.1.txt)
-- [ ] You have added tests for any code changes
-- [ ] You have updated the [CHANGES.md](https://github.com/cloudant/sync-android/blob/master/CHANGES.md)
-- [ ] You have completed the PR template below:
+- [ ] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](../DCO1.1.txt)
+- [ ] Added tests for code changes _or_ test/build only changes
+- [ ] Updated the change log file (`CHANGES.md`|`CHANGELOG.md`) _or_ test/build only changes
+- [ ] Completed the PR template below:
 
-## What
+## Description
 
-What was changed e.g.
->Corrected NPE when trying to read settings from a migrated index.
+<!--
```

Comment:
Do you still want the user to put a summary as well as the link to the issue? I personally would find this useful, and issue descriptions change over time and often don't capture the underlying bug (ie the GH issue is framed in "when I click x, y happens", but the description here should be "an NPE gets thrown because foo was set to baz")